src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 <InstallFeedback />
 
+## Install Fluent Bit output plugin [#fluent-bit]
+
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
```

Comment:
@paperclypse it's my understanding that a DaemonSet just makes sure a pod runs on certain nodes. The log forwarder sounds like a plug-in. Could you confirm with our devs on this?

Reply:
This content came from our dev's readme, so I assume that it's accurate.